frontend/components/Documents.js

```diff
@@ -1,11 +1,17 @@
 import React, {useEffect, useState} from "react";
 // import * as PropTypes from "prop-types";
 // import STYLES from "./Documents.module.scss";
-
+import {getCookie} from "../common";
 
 const Documents = () => {
 
-    const [docData, setDocData] = useState(null);
+    const [docData, setDocData] = useState([]);
+    const [newDocData, setNewDocData] = useState({
+        "author": "",
+        "title":"",
+        "date": null,
```

Comment:
We're getting some warnings in the console about the `date` property being `null` on default. We should change it to an empty string as with the others.

Reply:
I've updated the `date` property to be `year` in the React component and in the Django model and limit its value to be max 9999.